packages/synapse-react-client/src/components/SynapseChat/SynapseChat.tsx

```diff
@@ -66,26 +69,39 @@ export const SynapseChat: React.FunctionComponent<SynapseChatProps> = ({
   const [interactions, setInteractions] = useState<ChatInteraction[]>([])
   const [pendingInteraction, setPendingInteraction] =
     useState<ChatInteraction>()
+  const [currentlyProcessingJobId, setCurrentlyProcessingJobId] =
+    useState<string>()
   const [currentResponse, setCurrentResponse] = useState('')
-  const [currentProgressMessage, setCurrentProgressMessage] = useState<
-    string | undefined
-  >()
   const [currentResponseError, setCurrentResponseError] = useState('')
   // Keep track of the text that the user is currently typing into the textfield
   const [userChatTextfieldValue, setUserChatTextfieldValue] = useState('')
   const [initialMessageProcessed, setInitialMessageProcessed] = useState(false)
+  // const delay = (ms: number) => new Promise(resolve => setTimeout(resolve, ms))
```

Comment:
Remove, unless you still need these to test something

Reply:
```suggestion
```

Reply:
Is there an easier way to introduce some delay in this response during testing?  I need this delay to test the progress trace output.

Reply:
You could add the delay in mock service worker: https://v1.mswjs.io/docs/api/context/delay

Reply:
Tried adding this option to our asyncJobHandlers.generateAsyncJobHandlers() helper function, but with the delay it never returns.  Oh well, I'll just add it when I need to.

Reply:
Introducing in our asyncJobHandlers.generateAsyncJobHandlers() utility function caused the result to never return.  Not important, so I'm just going to clean up the "delay" code for now